### PR TITLE
Fix: #1785 Added covalent version attribute to Remote Executors

### PR DESCRIPTION
**Pull request by maskboyAvi**
Issue - #1785 

Added covalent version attribute to remote_executor.py

- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

File: covalent/executor/executor_plugins/remote_executor.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "poll_freq": 15,
     "remote_cache": ".cache/covalent",
     "credentials_file": "",
+    "covalent_version": "0.228.0-rc.0",
```
Comment (kessler-frost) [inline covalent/executor/executor_plugins/remote_executor.py:43]: Let's have the default as an empty string for now, we can later add a mechanism to determine the version automatically.

File: covalent/executor/executor_plugins/remote_executor.py [hunk 2]
```diff
@@ -58,13 +59,15 @@ def __init__(
         poll_freq: int = 15,
         remote_cache: str = "",
         credentials_file: str = "",
+        covalent_version: str = "0.228.0-rc.0",
```
Comment (kessler-frost) [inline covalent/executor/executor_plugins/remote_executor.py:62]: Here as well.

**Review by kessler-frost — CHANGES_REQUESTED** [review]
Thanks @maskboyAvi ! Mostly looks good, just small comments.

**Comment by maskboyAvi** [comment]
@kessler-frost  I have made the requested changes. Please have a look.

**Review by kessler-frost — APPROVED** [review]

**Comment by kessler-frost** [comment]
Thanks @maskboyAvi !

**Pull request merged**